src/Zicht/Bundle/FrameworkExtraBundle/Command/CreateStdLoggerTrait.php

```diff
@@ -0,0 +1,64 @@
+<?php
+/**
+ * @copyright Zicht Online <https://zicht.nl>
+ */
+
+namespace Zicht\Bundle\FrameworkExtraBundle\Command;
+
+use Monolog\Handler\FilterHandler;
+use Monolog\Handler\StreamHandler;
+use Monolog\Logger;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+trait CreateStdLoggerTrait
+{
+    /**
+     * Returns a logger configured to use STDOUT and STDERR depending on $input and $output settings
+     *
+     * Use this method to construct a logger that sends log messages to stdout or stderr.  This
+     * is especially useful for commands that are executed using crontab in combination with the
+     * linux tool cronic.
+     *
+     * Usage:
+     * ```php
+     * class ExampleCommand extends AbstractCommand
+     * {
+     *     use CreateStdLoggerTrait;
+     *
+     *     protected function execute(InputInterface $input, OutputInterface $output)
+     *     {
+     *         $logger = $this->createStdLogger($output, 'console');
+     *         $logger->debug(...);      // logs to stdout when -v or --verbose
+     *         $logger->info(...);       // logs to stdout
+     *         $logger->notice(...);     // logs to stdout
+     *         $logger->warning(...);    // logs to stdout
+     *         $logger->error(...);      // logs to stderr
+     *         $logger->critical(...);   // logs to stderr even when -q (quiet)
+     *         $logger->alert(...);      // logs to stderr even when -q (quiet)
+     *         $logger->emergency(...);  // logs to stderr even when -q (quiet)
+     *     }
+     * }
+     * ```
+     *
+     * @param OutputInterface $output
+     * @param string $name
+     * @return LoggerInterface
+     */
+    protected function createStdLogger(OutputInterface $output, string $name): LoggerInterface
+    {
+        $levelMap = [
+            OutputInterface::VERBOSITY_QUIET => Logger::CRITICAL,
+            OutputInterface::VERBOSITY_NORMAL => Logger::INFO,
+            OutputInterface::VERBOSITY_VERBOSE => Logger::DEBUG,
+            OutputInterface::VERBOSITY_VERY_VERBOSE => Logger::DEBUG,
+            OutputInterface::VERBOSITY_DEBUG => Logger::API,
```

Comment:
`Logger::API` is de (interne) Monolog API-versie. `Monolog API version. This is only bumped when API breaks are done and should follow the major version of the library`. 

Dit is geen loglevel uit de specs, ik denk niet dat we hier iets mee moeten/kunnen. Deze zou gewoon aan Logger::DEBUG kunnen hangen.